scripts/update/sonarr.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if dpkg -l | grep nzbdrone > /dev/null 2>&1; then
+    v2present=true
+fi
+if [[ -f /install/.sonarr.lock ]] && [[ $v2present == "true" ]]; then
+    #update lock file
+    rm /install/.sonarr.lock
+    touch /install/.sonarrv2-old.lock
+fi
+if [[ -f /install/.sonarrv2-old.lock ]]; then
+    #upgrade sonarr v2 to v3
+    box install sonarrv3
+fi
```

Comment:
This cannot be a forced migration. We have a manual migration for sonarr already, this is not going to pass. If someone chooses to stay on v2, so be it.

Reply:
no forced migration from v2 to v3 at this time